src/main/java/qna/domain/Question.java

```diff
@@ -26,6 +33,7 @@ public Question(Long id, User writeBy, String title, String contents) {
         this.writeBy = writeBy;
         this.title = title;
         this.contents = contents;
+        this.answers = new Answers(new ArrayList<>());
```

Comment:
```suggestion
        this.answers = new Answers();
```

와 같이 쓸 수 있으면 좋을 것 같아요 :)

Reply:
기본 생성자에서 초기화를 하면서 굳이 빈 리스트를 넣어주고 있었네요 😅
수정하겠습니다!